lib/qtcamconfig.cpp

```diff
@@ -118,9 +118,9 @@ QtCamConfig::QtCamConfig(QObject *parent) :
   d_ptr(new QtCamConfigPrivate) {
 
 #ifdef QT4
-  d_ptr->model = QtMobility::QSystemDeviceInfo().model().toLower();
+  d_ptr->model = QtMobility::QSystemDeviceInfo().productName().toLower();
```

Comment:
then you have to fix the packaging for them too ;)
Or promise to submit one PR per issue in the future and I will fix it for you :-)